meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -5,7 +5,7 @@ set -e
 DATA_PARTITION=/dev/disk/by-label/resin-data
 
 echo -n "Expand ext4 partition on $DATA_PARTITION... "
-e2fsck -y -f $DATA_PARTITION
+e2fsck -y $DATA_PARTITION
```

Comment:
I was under the impression that you had a task about not running resin-filesystem-expand at every boot. Was that a different task?